app/Http/Controllers/Admin/KeywordController.php

```diff
@@ -12,6 +12,13 @@ class KeywordController extends Controller
     private $keyword;
     private $genre;
 
+    public function __construct(Genre $genre, Keyword $keyword)
```

Comment:
I think we don't need this anymore since we are not using this in our controller, same with the `private $keyword` and `private $genre`

Reply:
I fixed that